ontology_api/model/model_service.py

```diff
@@ -75,3 +77,15 @@ def save_model_as_owl(self, model, model_id, path=None):
     def get_owl_from_model(self, model_id, path=None):
         model = self.__find_model_by_id(model_id)
         return self.save_model_as_owl(model, model_id, path)
+
+    def add_instance(self, class_id, model_id, model_in: MinimalInstanceModelIn):
```

Comment:
Please add description of this function. I don't get what this is supposed to do.... What class the `model` is? Does it have `classes` method? What is the result of this function? Is it setting something, or returning?